### DISCUSSION TITLE
znap-pull exits with return code 1

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [marlonrichert/zsh-snap#161] [General]
Every time I run `znap-pull` it exits with return code `1` and my prompt goes from green to red. I've looked at the code in [.znap-pull](https://github.com/marlonrichert/zsh-snap/blob/main/functions/.znap.pull) and to this piece of code:

```zsh
    if ! git diff --quiet @; then
      print -P '%F{red}dirty%f'
      return 1
    fi
```
I've not touched any of the packages that I'm using so I'm not sure why there would be difference between any local package and it's remote counterpart. I'll like to know exactly which package is causing this. Any help?

**Comment by marlonrichert** [comment]
#162
